src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -30,20 +30,19 @@
 using Xunit;
 using static Microsoft.CodeAnalysis.CommonDiagnosticAnalyzers;
 using static Roslyn.Test.Utilities.SharedResourceHelpers;
-using CscExe.Microsoft.CodeAnalysis.CSharp.CommandLine;
-using System.Security.Cryptography;
 
 namespace Microsoft.CodeAnalysis.CSharp.CommandLine.UnitTests
 {
     public class CommandLineTests : CSharpTestBase
     {
-        private static readonly string s_CSharpCompilerExecutable = typeof(Csc).GetTypeInfo().Assembly.Location;
+        private static readonly string s_CSharpCompilerExecutable = Path.Combine(
+            Path.GetDirectoryName(typeof(CommandLineTests).GetTypeInfo().Assembly.Location),
+            @"dependency\csc.exe");
```

Comment:
Do we have to worry about casing and backslash on different OS'es?

Reply:
Yes. This project doesn't run on unix yet but should get it right now so that it's easier to port later. 

---
In reply to: [172259424](https://github.com/dotnet/roslyn/pull/25213#discussion_r172259424) [](ancestors = 172259424)